CHANGELOG.md

```diff
@@ -1,81 +0,0 @@
-## Changelog
```

Comment:
We should leave the changelog here as this is the only place we keep an external changelog. The documentation also references it multiple times.